docs/proposals/algorithms/multi-edge-inference/proposal.md

```diff
@@ -0,0 +1,104 @@
+# Multi-task Joint Inference for Ianvs
+
+## Motivation
+Nowadays,  the inference of lifelong learning is somehow a single task approach, which will first vote for the task suited for the sample, and then the model trained for this task will be used for inference. This approach works well when the model for the task is easy to train. However, for the perception of mobile agents, more complex models are usually used. Besides,  as shown in figure 1, an empirical study concluded that no single model maintains the best accuracy even for one second. So we think the joint inference will be better.
+
+<img src="Fig1_Heatmap of normalized mAP on a clip of 1 second.png" width="400" />
+
+​														Fig 1. Heatmap of normalized mAP on a clip of 1 second
+
+
+We have noticed that autonomous driving is one of the important applications of edge AI. We consider the image object detection task on moving vehicles as the typical online model inference scenario. Recently, how to cooperate with resources on the edge and cloud to provide support for autonomous driving applications has become an important topic.  
+
+### Goals
+
+The goals of the multi-task joint inference for Ianvs are:
+
+- Adding an example named multi-task joint inference in Ianvs 
+  - choose tasks to realize joint inference according to various scenarios dynamically
+    - the characteristics of vehicle movement, the scenarios faced by autonomous vehicles are complicated
+  - fuse the output of multi tasks
```

Comment:
In the example, except algorithm-wise procedures like ``choose tasks ...`` and ``fuse the output...``, the goals should include scenario description of the autopilot dataset, a multi-task joint inference algorithm, and test report of this algorithm.